lib/rules/selector-combinator-blacklist/README.md

```diff
@@ -0,0 +1,50 @@
+# selector-combinator-blacklist
+
+Specify a blacklist of disallowed combinators.
+
+```css
+  a + b {}
+/** ↑
+ * These combinators */
+```
+
+This rule ignores [reference combinators](https://www.w3.org/TR/selectors4/#idref-combinators) e.g. `/for/`.
+
+## Options
+
+`array|string|regex`: `["array", "of", "unprefixed", "pseudo-elements" or "regex"]|"pseudo-element"|/regex/`
+
+Given:
+
+```js
+[" ", "/^>/"]
+```
+
+The following patterns are considered violations:
+
+```css
+a b {}
+```
+
+```css
+a > b {}
+```
+
+```css
+a >>> b {}
```

Comment:
It's not a standard combinator. Maybe it's better not to add it to readme?

Reply:
Good point